### PR TITLE
[Method References 2] LPS-171231 Convert method reference to ServiceTrackerMap in module portal-workflow-kaleo-runtime-impl | part 3

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:workflow

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

**Comment by W2jx** [comment]
no conflicts(2022/12/16 8:47)

**Comment by W2jx** [comment]
no conflicts(12/19 9:48)

**Comment by W2jx** [comment]
no conflicts（12/20 11:57）

**Comment by W2jx** [comment]
no conflicts(12/21 17:05)

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/assignment/MultiLanguageKaleoTaskAssignmentSelector.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public Collection<KaleoTaskAssignment> getKaleoTaskAssignments(
 		KaleoTaskAssignmentSelector kaleoTaskAssignmentSelector = null;
 
 		if (assigneeClassName.equals(ResourceAction.class.getName())) {
-			kaleoTaskAssignmentSelector = _kaleoTaskAssignmentSelectors.get(
+			kaleoTaskAssignmentSelector = _serviceTrackerMap.getService(
```
Comment (tinatian) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/assignment/MultiLanguageKaleoTaskAssignmentSelector.java:63]: @W2jx The code before your change does not seem right, how could you retrieve something by "assigneeClassName" when the key is script language.

**Comment by tinatian** [comment]
@W2jx Please see my comment above and also changes at https://github.com/liferay-core-infra/liferay-portal/pull/2812, review your code here and resend.

Thanks.
Tina

**Pull request closed without merging**